sriov_antrea/sriov_antrea_ci_start.sh

```diff
@@ -75,9 +75,13 @@ EOF
     if [[ -z "${ANTREA_CNI_PR}${ANTREA_CNI_BRANCH}" ]];then
         change_image_name $ANTREA_CNI_HARBOR_IMAGE antrea/antrea-ubuntu
     fi
-    
+
+    cp $WORKSPACE/antrea-cni/build/yamls/antrea.yml $ARTIFACTS
+
+    sed -i "s;\(^ *image\:\).*;\1 antrea/antrea-ubuntu;" $ARTIFACTS/antrea.yml
+ 
     if [[ -z "$(grep hw-offload $WORKSPACE/antrea-cni/build/yamls/antrea.yml)" ]];then
```

Comment:
nit: I think it is more clear to do  grep in $ARTIFACTS folder. 
Maybe it will be better to use --hw-offload as pattern for grep?
if condition can be simplified to remove subshell usage
```
if ! grep -q  --hw-offload $ARTIFACTS/antrea-cni/build/yamls/antrea.yml; then
...
fi
```